draft-ietf-trans-rfc6962-bis.md

```diff
@@ -2391,3 +2393,87 @@ SCTs:
 
 * Sign that TBSCertificate (which now contains v1 and v2 SCTs) to issue the
   final X.509 certificate.
+
+# An ASN.1 Module (Informative)
+
+The following ASN.1 module may be useful to implementors.
+
+~~~~~~~~~~~
+
+CertificateTransparencyV2Module-2021
+ -- { OID Needed, but no point in using a short one }
+DEFINITIONS IMPLICIT TAGS ::= BEGIN
+
+-- EXPORTS ALL --
+
+IMPORTS
+  EXTENSION
+  FROM PKIX-CommonTypes-2009 -- RFC 5912
+    { iso(1) identified-organization(3) dod(6) internet(1)
+      security(5) mechanisms(5) pkix(7) id-mod(0)
+      id-mod-pkixCommon-02(57) }
+
+  CONTENT-TYPE
+  FROM CryptographicMessageSyntax-2010  -- RFC 6268
+    { iso(1) member-body(2) us(840) rsadsi(113549) pkcs(1)
+      pkcs-9(9) smime(16) modules(0) id-mod-cms-2009(58) }
+
+  TBSCertificate
+  FROM PKIX1Explicit-2009 -- RFC 5912
+    { iso(1) identified-organization(3) dod(6) internet(1)
+      security(5) mechanisms(5) pkix(7) id-mod(0)
+      id-mod-pkix1-explicit-02(51) }
+;
+
+--
+-- Section 3.2.  Precertificates
+--
+
+ct-tbsCertificate CONTENT-TYPE ::= {
+  TYPE TBSCertificate
+  IDENTIFIED BY id-ct-tbsCertificate }
+
+id-ct-tbsCertificate OBJECT IDENTIFIER ::= { 1 3 101 78 }
+
+--
+-- Section 7.1.  Transparency Information X.509v3 Extension
+--
+
+ext-transparencyInfo EXTENSION ::= {
+   SYNTAX TransparencyInformationSyntax
+   IDENTIFIED BY id-ce-transparencyInfo
+   CRITICALITY { FALSE } }
+
+id-ce-transparencyInfo OBJECT IDENTIFIER ::= { 1 3 101 75 }
+
+TransparencyInformationSyntax ::= OCTET STRING
+
+--
+-- Section 7.1.1.  OCSP Response Extension
+--
+
+ext-ocsp-transparencyInfo EXTENSION ::= {
+   SYNTAX TransparencyInformationSyntax
+   IDENTIFIED BY id-pkix-ocsp-transparencyInfo
+   CRITICALITY { FALSE } }
+
+id-pkix-ocsp-transparencyInfo OBJECT IDENTIFIER ::=
+   id-ce-transparencyInfo
+
+--
+-- Section 8.1.2.  Reconstructing the TBSCertificate
+--
+
+ext-embeddedSCT EXTENSION ::= {
```

Comment:
To make it absolutely clear which CT version this extension belongs to, WDYT about naming this extension `ext-embeddedSCT-CTv1` and the corresponding OID `id-ce-embeddedSCT-CTv1`?